File: travel-page-category-interface/src/main/java/com/itheima/page/CategoryService.java
package com.itheima.page;


import com.itheima.pojo.TabCategory;

import java.util.List;


public interface CategoryService {

    //查询所有
    public List<TabCategory> findAll();

    //根据id查询旅游分类列表
    public List<TabCategory> findById(Integer cid);
}

File: travel-page-category-web/src/main/webapp/js/service/cateGoryService.js
 //服务层
 app.service('categoryService',function($http){
    //读取列表数据绑定到表单中
     this.findAll=function () {
         return $http.get('category/findAll.do');
     }
// ---------------------------------------


 // -------------------------------------

   //根据id查询导航列表
     this.findById=function(cid){
         return $http.get('category/findById.do?cid='+cid)
     }

 });